spec/api-system-preferences-spec.js

```diff
@@ -170,17 +172,17 @@ describe('systemPreferences module', () => {
 
   describe('systemPreferences.isInvertedColorScheme()', () => {
     it('returns a boolean', () => {
-      assert.strictEqual(typeof systemPreferences.isInvertedColorScheme(), 'boolean')
+      expect(systemPreferences.isInvertedColorScheme()).to.be.a('boolean')
     })
   })
 
   describe('systemPreferences.getAnimationSettings()', () => {
     it('returns an object with all properties', () => {
       const settings = systemPreferences.getAnimationSettings()
-      assert.strictEqual(typeof settings, 'object')
-      assert.strictEqual(typeof settings.shouldRenderRichAnimation, 'boolean')
-      assert.strictEqual(typeof settings.scrollAnimationsEnabledBySystem, 'boolean')
-      assert.strictEqual(typeof settings.prefersReducedMotion, 'boolean')
+      expect(settings).to.be.an('object')
+      expect(settings.shouldRenderRichAnimation).to.be.a('boolean')
```

Comment:
slightly better as `expect(settings).to.have.a.property('shouldRenderRichAnimation').that.is.a('boolean')`